pkg/deploy/api/types.go

```diff
@@ -257,6 +257,8 @@ type DeploymentConfig struct {
 	// Details are the reasons for the update to this deployment config.
 	// This could be based on a change made by the user or caused by an automatic trigger
 	Details *DeploymentDetails
+	// Replicas is the total number of pods targeted by this deploymentConfig.
+	Replicas int
```

Comment:
> I think I missed why we can't bring the Status field internal.

I am wondering about this. Our versioned dcs have status but not the internal one. Is there any reason why?